Utilities/DB.php

```diff
@@ -130,4 +126,10 @@ public function fillCatImg()
         $sql = $dbconnect->prepare('INSERT INTO `img` (img_src, breed_id) VALUES ' . $sqlString . ';');
         $sql->execute();
     }
-}
\ No newline at end of file
+}
+# Make a new object and call the methods
+$DBobject = new DB();
+$breeds = $DBobject->getCatBreed();
+$DBobject->fillCatBreedToDB($breeds);
+$catImgArcArray = $DBobject->getCatImg($breeds);
+$DBobject->fillCatImg($breeds, $catImgArcArray);
```

Comment:
typo `$catImgArcArray` -> `$catImgSrcArray`

Reply:
Tom did it! fixed !